storybook/stories/API/cartesian/Scatter.stories.tsx

```diff
@@ -526,10 +483,6 @@ export default {
     ...EventHandlers,
     ...AnimationProps,
     ...ReactiveProps,
-    ...OtherProps,
-    ...InternalProps,
-    // Deprecated
-    dangerouslySetInnerHTML: { table: { category: 'Deprecated' }, hide: true, disable: true },
```

Comment:
these no longer get spread on the element?

Reply:
These were never external props, whatever came from the outside was overwritten by the generator. It happened to be documented as "Internal, do not use" because Recharts typescript types did not handle the difference between real props, and props coming from cloning.

Since we no longer have cloning we also don't have the type problem so we can remove this stuff from the docs.

Reply:
I thought they ended up on the high level SVG. Anyways, as long as they don't exist in the types then all good

Reply:
Speaking of dangerouslySetInnerHTML in particular, let's remove it in 3.x too? It's marked as deprecated.

Reply:
yeah absolutely. Ideally, I'd only want to include implicit SVG props that are actually useful. Thats easier said than done though since there are so many